materialize-bigquery/.snapshots/TestSpecification

```diff
@@ -30,10 +30,7 @@
         "type": "string"
       },
       "credentials_json": {
-        "type": "string",
-        "media": {
-          "binaryEncoding": "base64"
-        }
+        "type": "string"
```

Comment:
I just realized this was also missing the `secret` annotation. This should also have `"secret": true`, which will cause the value to be encrypted. I just added some [docs on that](https://github.com/estuary/connectors/blob/a161ad0ebb02fbbfe502466f6d8f6c8f85063f05/config_schema_guidelines.md#use-secret-annotation-for-all-sensitive-fields), for reference.

Reply:
I'll do it in another PR that I'll start tomorrow. Since this is still using the sqlDriver abstraction, I'm not sure how simple it is to do. I need to research it.

[Filed an issue for it](https://github.com/estuary/connectors/issues/206)